### PR TITLE
Rename and add interfaces

### PR DESCRIPTION
**Pull request by OnedgeLee**
SSIA

### CODE REVIEW

File: unirec/models/layers.py [hunk 1]
```diff
@@ -0,0 +1,498 @@
...
+
+        self.filters = filters
+
+        for i in range(len(filters)):
+            if i > 0:
+                in_channels = filters[i - 1]
+            setattr(
+                self,
+                "conv2d_{}".format(i),
+                nn.Conv2d(in_channels, filters[i], kernels[i], padding="same"),
```
[the remark this answers is not in the document]
Comment (OnedgeLee) [reply unirec/models/layers.py]: As I know, latest version of PyTorch supports padding="same" on nn.Conv2d.
